src/mongocrypt.h

```diff
@@ -25,14 +25,18 @@
 
 #include "mongocrypt-export.h"
 #include "mongocrypt-compat.h"
-#include "mongocrypt-config.h"
 
-/**
- * @def MONGOCRYPT_VERSION
- * The version string describing libmongocrypt.
- * Has the form x.y.z-<pre>+<date>+git<sha>.
- */
-#define MONGOCRYPT_VERSION "@MONGOCRYPT_BUILD_VERSION@"
+/* clang-format off */
+#ifndef __has_include
+   #include "mongocrypt-config.h"
+#else
+   #if __has_include("mongocrypt-config.h")
+      #include "mongocrypt-config.h"
+   #else
+      #error "No 'mongocrypt-config.h' header is available. This must be generated in order to use libmongocrypt"
```

Comment:
Confirming, is the purpose of detecting and using `__has_include` solely to provide a better error message given ``"mongocrypt-config.h"` cannot be found?